src/accessibility/sonification/index.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+var c2m = require('chart2music');
+var Fx = require('../../components/fx');
+var Lib = require('../../lib');
+
+var codecs = require('./all_codecs').codecs;
+
+function initC2M(gd, defaultConfig) {
+    // TODO: should this be Lib.getGraphDiv()?
+    // TODO what is there besides a fullReset?
```

Comment:
What does this comment mean? @ayjayt

Reply:
Hi sorry! I just fixed my github notifications today

Line 10: there is a function `Lib.getGraphDiv()` that seems to coerce `gd` to be a "real" `gd` but I'm not sure when its needed.
Line 20: Me not knowing the different "levels" of plotly's rendering functions (fullReset being a full-rerender, but there are shortcuts) had previously caused bugs, so my basic question is "When do we actually need to re-initialize C2M besides a fullReset?"

Reply:
`getGraphDiv` is for functions to be called by end users, so that they can pass in the ID of their graph div as a string, as an alternative to passing in the DOM element directly. Functions called only by our code do not need this step. Which kind is `initC2M`?